src/index.d.ts

```diff
@@ -134,7 +134,7 @@ export interface ForceGraphGenericInstance<ChainableInstance> {
 
   // Force engine (d3-force) configuration
   dagMode(): DagMode;
-  dagMode(mode: DagMode): ChainableInstance;
+  dagMode(mode: DagMode | false): ChainableInstance;
```

Comment:
You can indicate DAG mode disabling by passing any falsy value onto `dagMode`, not just `false` itself. So it would be more appropriate to indicate that in the types (something that includes also `undefined`, `null` or even empty string), but afaik TS doesn't have a built-in `Falsy` type, does it? Wondering what's the most canonical and idiomatic way to indicate this.

Also, whatever type extensions we put on this lin, need to go as well on the previous line L36, as return type of the read method.

Reply:
> TS doesn't have a built-in Falsy type

Yes. And because empty string `''` would be confusing. Using `undefined` may conflict with `dagMode(): DagMode;`. So the choice is between `false` or `null`. So I put `false` here.

If taking L136 into consideration, I would prefer `null` over `false`, which is also the default value returned by [Kapsule](https://github.com/vasturiano/kapsule#default).